cmd/service/main.go

```diff
@@ -129,6 +136,15 @@ func main() {
 		}); err != nil {
 			log.Fatalf("could not create the email service: %v", err)
 		}
+		// load email templates if the path is set
+		if emailTemplatesPath != "" {
+			if err := mailtemplates.Load(emailTemplatesPath); err != nil {
```

Comment:
I do not understand how the loaded templates here remain accesible in the rest of the execution. `mailtemplates` is a pacakge and not an instance of somethig,

Reply:
The `mailtemplates` package includes the `AvailableTemplates` package variable (which is a map) that includes the reference to the templates files. This variable is initialized when someone execs `Load` public function and it is available for any other package while the program is running.